client/src/app/components/feed.jsx

```diff
@@ -6,6 +6,9 @@ var { Colors, Spacing, Typography } = mui.Styles;
 var CardList = require('./card-list.jsx');
 var FavoriteList = require('./favorite-list.jsx');
```

Comment:
CardList and FavoriteList are not used any more.
They should be removed in next patch.